server/playground/index.js

```diff
@@ -0,0 +1,33 @@
+const buildSchema = require("@aerogear/data-sync-gql-core");
+const { makeExecutableSchema } = require("graphql-tools");
+const { ApolloServer } = require("apollo-server-express");
+const { log } = require("../logger");
+const models = require("../models");
+
+async function connectDataSources(dataSources) {
+    log.info("Connecting data sources");
+    Object.keys(dataSources).forEach(async key => {
+        const dataSource = dataSources[key];
+        try {
+            await dataSource.connect();
+        } catch (error) {
+            log.error(`Error while connecting datasource with key ${key}`);
+            log.error(error);
+            throw (error);
+        }
+    });
+}
+
+
+async function ApplyExecutableMiddleware(app) {
+    // read default schema
+    // Attach middleware to app
+    const { schema, dataSources } = await buildSchema(models, null, null, makeExecutableSchema);
```

Comment:
My assumption is that this is going to read "default" schema. 
We could allow users to pass string so the schema will be configurable in core?
Could models be embedded into core (they are there already anyway) so end user api will be like: `createMiddlewareFor({schema:"default"})`

This aproach is working but TBH merging it this way will introduce tech debt.